theme-local/pointsofarrival/css/style.css

```diff
@@ -1289,7 +1313,7 @@ div[id^="gallery"] {
 }
 
 #galleryethpe {
-    background-image: url(https://images.is.ed.ac.uk/luna/servlet/iiif/uoeart~2~2~73247~166273/167,213,5092,4725/1024,/0/default.jpg);
+    background-image: url(https://test.collections.ed.ac.uk/theme/pointsofarrival/images/index-links/resources.png);
```

Comment:
IMO images used as links should not be backgrounds. Having images as backgrounds in Skylight is especially troublesome as it's not php rendered.

Move src to a nested img element?

Same for below?